adhocracy4/images/services.py

```diff
@@ -3,6 +3,7 @@
 
 def delete_images(imagefields):
     for imagefield in imagefields:
+        if not imagefield: continue
```

Comment:
How can that happen? Is that the case if the image has not been set?

And why is the continue on the same line as the if? :smile: